src/main/java/com/google/sps/servlets/NewScheduledInterviewServlet.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import com.google.sps.data.DatastoreScheduledInterviewDao;
+import com.google.sps.data.ScheduledInterview;
+import com.google.sps.data.ScheduledInterviewDao;
+import com.google.sps.data.TimeRange;
+
+@WebServlet("/new-scheduled-interview")
+public class NewScheduledInterviewServlet extends HttpServlet {
+
+  private ScheduledInterviewDao scheduledInterviewDao;
+
+  @Override
+  public void init() {
+    init(new DatastoreScheduledInterviewDao());
+  }
+
+  // TODO: add a FakeScheduledInterviewDao class so this will become useful
+  public void init(ScheduledInterviewDao scheduledInterviewDao) {
+    this.scheduledInterviewDao = scheduledInterviewDao;
+  }
+
+  // Sends the request's contents to Datastore in the form of a new ScheduledInterview.
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/MM/yyyy");
+    ScheduledInterview scheduledInterview =
+        ScheduledInterview.create(
+            0,
+            new TimeRange(
```

Comment:
Datastore can't represent TimeRange. What l'm doing with Availability is converting TimeRange's lnstants to longs using https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toEpochMilli--

and storing longs in Datastore. 
l'm thinking on the front end we should have these longs stored in some data- attribute in the HTML and we just pass them in as longs here.
lt might honestly be more worth it to get some of the front end set up first, so we know what's being passed in exactly.

Reply:
I'm currently storing the instants as strings in Datastore via the create method in DatastoreScheduledInterview.java but I can change them to longs instead

Reply:
Hm yeah it'd be better if it matched with Availability entities